Common/Data/UniverseSelection/Universe.cs

```diff
@@ -227,6 +243,33 @@ public virtual IEnumerable<SubscriptionRequest> GetSubscriptionRequests(Security
                 );
         }
 
+
+        /// <summary>
+        /// Gets the subscription requests to be added for the specified security
+        /// </summary>
+        /// <param name="security">The security to get subscriptions for</param>
+        /// <param name="currentTimeUtc">The current time in utc. This is the frontier time of the algorithm</param>
+        /// <param name="maximumEndTimeUtc">The max end time</param>
+        /// <param name="subscriptionService">Instance which implements <see cref="ISubscriptionDataConfigService"/> interface</param>
+        /// <returns>All subscriptions required by this security</returns>
+        public virtual IEnumerable<SubscriptionRequest> GetSubscriptionRequests(Security security,
+            DateTime currentTimeUtc,
+            DateTime maximumEndTimeUtc,
+            ISubscriptionDataConfigService subscriptionService)
+        {
+
+            var result = subscriptionService.Add(security.Symbol,
+                                                 UniverseSettings.Resolution,
+                                                 UniverseSettings.FillForward,
+                                                 UniverseSettings.ExtendedMarketHours);
+            return result.Select(config => new SubscriptionRequest(isUniverseSubscription: false,
```

Comment:
no need to change, but I tend to prefer left-aligned code as it prevents us ending up too far to the right
``` csharp
var result = subscriptionService.Add(
	security.Symbol,
	UniverseSettings.Resolution,
	UniverseSettings.FillForward,
	UniverseSettings.ExtendedMarketHours
);
	
return result.Select(config => new SubscriptionRequest(
	isUniverseSubscription: false,
	universe: this,
	security: security,
	configuration: config,
	startTimeUtc: currentTimeUtc,
	endTimeUtc: maximumEndTimeUtc
));
```

I find this style leads to more alignment between constructs, vs aligning with a parenthesis leads to various things not being aligned (I know, I'm picky with code formatting :P )

Reply:
No worries, I'll re format it :)